src/components/Home/Home.js

```diff
@@ -7,12 +7,28 @@ import './Home.css';
 
 export default function Home() {
   // add useState calls here for title, subtitle, font, align, and text
+  const [title, setTitle] = useState('');
+  const [subtitle, setSubtitle] = useState('');
+  const [font, setFont] = useState([]);
+  const [align, setAlign] = useState([]);
+  const [text, setText] = useState('');
 
   return (
     <main>
       {/* pass the state variables as props to the presentational components */}
-      <Preview />
-      <Editor />
+      <Preview title={title} subtitle={subtitle} font={font} align={align} text={text} />
+      <Editor
+        title={title}
+        setTitle={setTitle}
+        subtitle={subtitle}
+        setSubtitle={setSubtitle}
+        font={font}
+        setFont={setFont}
+        align={align}
+        setAlign={setAlign}
+        text={text}
+        setText={setText}
+      />
```

Comment:
It doesn't look like you use `align` or `font` in the definition of your `<Editor>` component, so you don't need to pass them here.

Reply:
Awesome, thanks Vic!